extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js

```diff
@@ -0,0 +1,47 @@
+require(
+	[
+		'jquery', 'wikia.window', 'wikia.delayedhover', 'wikia.globalNavigationDropdowns'
+	],
+	function ($, w, delayedHover, dropdowns) {
+		'use strict';
+
+		var $entryPoint,
+			$exploreLink;
+
+		/**
+		 * @desc Handle click on entry point for logged in users.
+		 * Second click on entry point for logged in users is redirecting to user profile page.
+		 * This method should be removed after we unify the ux for anon and logged in.
+		 * @param {Event} event
+		 */
+		function onEntryPointClick(event) {
+			var $this = $(event.target);
+			event.preventDefault();
+			event.stopImmediatePropagation();
+			if (this.$entryPoint.hasClass('active')) {
+				w.location = $this.attr('href');
+			} else {
+				dropdowns.openDropdown.call(this.$entryPoint.get(0));
+			}
+		}
+
+		$(function () {
+			$entryPoint = $('#exploreWikiaEntryPoint');
```

Comment:
please use consistent way to access this function variables.
use `this.$entryPoint` or `$entryPoint` in whole file